arm_wiki/Alternate-Install-TrueNAS.md

```diff
@@ -42,22 +42,28 @@ Prior versions of TrueNAS Scale have an issue with GPU Allocation, which Cobia f
    Continue down to the ‘Resource Reservation’ section and into the ‘GPU Configuration’
    If you have a supported GPU, allocate at least a single one of the GPUs to enable the NVENC(Geforce) or other encodings, it will show you multiple GPUs that you can allocate, choose at least one.
       - For NVIDIA GPUs, not only do need to allocate your GPU, but you must also add a pair of variables in the ‘Container Environment Variables’ section:
-        - name: ‘NVIDIA_VISIBLE_DEVICES’ value: ‘all’
-        - name: ‘NVIDIA_DRIVER_CAPABILITIES’ value: ‘all’
+        - name: `NVIDIA_VISIBLE_DEVICES` value: `all`
+        - name: `NVIDIA_DRIVER_CAPABILITIES` value: `all`
 
-6. (Optional) Enable Web Portal. Not a needed step, but this will create a button on the ARM app in TrueNAS Scale to allow quick access to the ARM Web GUI.
+6. (Optional) Enable Web Portal. Not a necessary step, but this will create a button on the ARM app in TrueNAS Scale
+to allow quick access to the ARM Web GUI.
+Web Portal access requires enabling of port forwarding. Under 'Enable WebUI Porta' configure port forwarding
```

Comment:
Sorry, I should have clarified on my last comment. These are 2 separate categories in the settings:
![image](https://github.com/user-attachments/assets/e7cb4a1e-2418-4a2a-9cd8-fbc347c555e7)

The Port Forwarding is configured like this (and required):
![image](https://github.com/user-attachments/assets/7dc8c37f-a2d0-4d86-b748-f3a8a2150e6a)

The Portal is then activated here (and optional):
![image](https://github.com/user-attachments/assets/254a5164-fe6a-4653-b5fb-293f9e16ddbf)